book/tables/adverse-events/aet02_smq.qmd

```diff
@@ -135,27 +137,32 @@ lyt <- basic_table(show_colcounts = TRUE) %>%
     label_pos = "topleft",
     split_label = "Standardized MedDRA Query"
   ) %>%
-  summarize_num_patients(
+  analyze_num_patients(
     var = "USUBJID",
     .stats = c("unique", "nonunique"),
     .labels = c(
       unique = "Total number of patients with at least one adverse event",
       nonunique = "Total number of events"
-    )
+    ),
+    show_labels = "hidden"
   ) %>%
   count_occurrences(
     vars = "AEDECOD",
-    .indent_mods = -1L,
     drop = FALSE
```

Comment:
I think this will do for now. @shajoezhu this should solve the issue put in `rtables` and `tern`, right @edelarua?

Reply:
Potentially solving (with a workaround):
- https://github.com/insightsengineering/rtables/issues/699
- https://github.com/insightsengineering/tern/issues/996
- (maybe) https://github.com/insightsengineering/teal.modules.clinical/issues/790